CMakeLists.txt

```diff
@@ -44,7 +44,7 @@ if(REAL_TEST)
 endif()
 
 #Google Benchmark, turn on/off with -DREAL_BENCH=ON or OFF
-option(REAL_BENCH "Build benchmarks" OFF)
+option(REAL_BENCH "Build benchmarks" ON)
```

Comment:
I think it will be nice to default to OFF so we don't run it in every build we do while developing. 
Alternatively, add the parameter to override it to off in .travis